src/componants/Comments.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react"
+import { getCommentsByArticleId } from "../../api"
+import CommentCard from "./CommentCard"
+
+function Comments (props){
+
+  const [comments, setComments] = useState([])
+  const {article_id} = props
```

Comment:
What would you expect to see whilst waiting for the comments to be returned? What if it failed?